### PR TITLE
Create principalComponentAnalysis.jsonld

### PR DESCRIPTION
**Pull request by lzehl**
@apdavison is the definition okay? related to #335

### CODE REVIEW

**Review by UlrikeS91 — APPROVED** [review]

**Comment by UlrikeS91** [comment]
I think the definition is fine (and understandable), but I'm not an expert. Maybe we want to wait for @apdavison review before merging?

**Comment by lzehl** [comment]
@UlrikeS91  I think for now that definition is okay. I'm anyway planning a hackathon where we can tackle definitions of analysis techniques with experts. For now they should just not be completely wrong

**Review by apdavison — APPROVED** [review]

**Pull request merged**
